doc/api/debugger.md

```diff
@@ -179,12 +183,13 @@ process or via URI reference to the listening debugger:
 * `node debug <URI>` - Connects to the process via the URI such as
 localhost:5858
 
-## V8 Inspector Integration for Node.js
+### V8 Inspector Integration for Node.js
 
 **NOTE: This is an experimental feature.**
```

Comment:
I wonder if we should remove the "experimental feature" note, especially if we're pointing people to it as an alternative to the deprecated debugger. That doesn't have to happen in this PR (although it could if there's consensus that it should happen).